dev/com.ibm.ws.org.apache.cxf.cxf.rt.rs.client.3.2/bnd.overrides

```diff
@@ -13,6 +13,7 @@ Import-Package: \
   !org.springframework.*,\
   !javax.cache.*,\
   !org.osgi.service.blueprint.*,\
+  com.ibm.ws.concurrent.mp.*;resolution:=optional,\
```

Comment:
I'm confused by this line because com.ibm.ws.concurrent.mp is a private package (not exported) and so will never be available for import.  This won't fail, because the import is optional.  The line just doesn't seem to serve any purpose.